### DISCUSSION TITLE
暂时的解决方法!  有需要的可以参考.

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#324] [Announcements]
有一位用户的建议. 暂时的解决方法. 您们试试看.

如下:

删除，蓝灯安装路径下的global.yaml文件
C:\Users\Administrator.DESKTOP-GINF5UQ\AppData\Roaming\Lantern
这样可以正常打开，但是是免费版本的，你再正常激活，你这次只要不关闭蓝灯就能使用
应该蓝灯的认证出问题了

---
我们会尽快修复的. 非常抱歉. 谢谢你们的耐心等待.

**Comment by rain526** [comment]
之前win11 突然无法打开，点击图标没反应，管理员运行也不行，删除了global文件实测可以打开了。
